opengrok-indexer/src/main/java/org/opengrok/indexer/web/messages/MessagesContainer.java

```diff
@@ -297,7 +323,8 @@ public void serialize(
                     final SerializerProvider serializerProvider
             ) throws IOException {
                 if (instant != null) {
-                    jsonGenerator.writeString(instant.toString());
+                    DateFormat formatter = DateFormat.getDateTimeInstance(DateFormat.LONG, DateFormat.LONG, Locale.ROOT);
+                    jsonGenerator.writeString(formatter.format(Date.from(instant)));
```

Comment:
nb i found `new ISO8601DateFormat()` from jackson databind

Reply:
It's deprecated: https://github.com/FasterXML/jackson-databind/issues/1786

Reply:
Also, I find ISO 8601 date format not very readable for humans.

Reply:
ok